cdap-proto/src/main/java/io/cdap/cdap/proto/security/StandardPermission.java

```diff
@@ -52,6 +52,10 @@ public boolean isCheckedOnParent() {
    * Permission to delete specific entity.
    */
   DELETE,
+  /**
+   * Permission to tether other clusters with this CDAP instance
+   */
+  TETHER,
```

Comment:
Why is it StandardPermission? StandardPermission is for permission that is available for any resource. This looks like a special permission for some specific resource. Which resource is it going to be checked against?

Reply:
It is specifically for CDAP instance. Created a new class `InstancePermission`.